src/simtools/ray_tracing/ray_tracing.py

```diff
@@ -636,8 +636,9 @@ def _generate_file_name(
         self, file_type, suffix, off_axis_angle=None, mirror_number=None, extra_label=None
     ):
         """Generate file name for output files."""
+        file_type_prefix = file_type if file_type == "ray_tracing" else f"ray_tracing_{file_type}"
```

Comment:
Can you help me with this line? I don't understand it.

Reply:
Sometimes we get the file type as `ray_tracing` (and that would be the prefix of the file) and sometimes we just get `photons`. In order to make sure the file prefix always starts with `ray_tracing`, avoiding redundancy when `file_type` is already `ray_tracing`, I added this line. It's not pretty, but it's the shortest way I found how to do it.

If you think it's clearer, I can replace it with 
```
file_type_prefix = f"ray_tracing_{file_type}" if file_type != "ray_tracing" else file_type
```

Reply:
OK, thanks - this is clear now.